DDMP.tex

```diff
@@ -0,0 +1,37 @@
+\section{DM Discussion and Decision Making Process}
```

Comment:
_If so,_ we can move to a model where https://developer.lsst.io highlights DM policies that affect DM staff on a day-to-day basis and links to the policy text in the appropriate LDM. One of the great successes of developer.lsst.io is that we _do_ read it and link to it frequently, making DM policies much more accessible than they were before in DocuShare or Confluence.